addons/dark-www/experimental_scratchwww.css

```diff
@@ -235,7 +235,7 @@ body:not(.sa-body-editor) .join-flow-privacy-message,
 .thumbnail .thumbnail-remixes::before,
 .thumbnail .thumbnail-views::before,
 .ttt-item img,
-.tabs button:not(.active) .tab-icon,
+.tabs button:not(.active) .tab-icon:not(.sa-search-profile-profilepic),
```

Comment:
`:not` increases the specificity of the selector. Adding a rule with `filter: none` to `addons/search-profile/style.css` is probably a better option:
```css
.tabs button:not(.active) .tab-icon.sa-search-profile-profilepic {
  filter: none;
}
```